src/NugetFeedDeploymentRepository/Logger.cs

```diff
@@ -0,0 +1,94 @@
+﻿using NuGet.Common;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Etg.Yams.NuGet.Storage
+{
+    public class Logger : ILogger
```

Comment:
This is an implementation of the adapter pattern to adapt Trace to the ILogger interface. It would be good if the name can reflect that (e.g. TraceToNuGetILoggerAdapter).

Reply:
Renaming to `TraceLogger`